xclim/ensembles/_robustness.py

```diff
@@ -228,6 +265,15 @@ def change_significance(
         f"Significant change was tested with test {test} with parameters {kwargs_str}."
     )
     das = {"fut": fut} if ref is None else {"fut": fut, "ref": ref}
+    changed.attrs.update(
+        description="Change significance. " + test_str,
+        units="",
+        test=str(test),
+        histroy=update_history(
+            f"change_significance(fut=fut, ref=ref, test={test}, {kwargs_str})",
```

Comment:
You could use `xclim.core.formatting.gen_call_string` here I think!

Reply:
Haha, I just realized we didn't use it further down... Woups, I can change that.